.circleci/config.yml

```diff
@@ -73,28 +81,24 @@ jobs:
           command: |
             cd examples/angular-cli
             yarn build-storybook
-
       - run:
-          name: "Run react kitchen-sink"
+          name: "Run react kitchen-sink (smoke test)"
           command: |
             cd examples/cra-kitchen-sink
-            yarn storybook
-          background: true
+            yarn storybook --smoke-test
       - run:
-          name: "Run vue kitchen-sink"
+          name: "Run vue kitchen-sink (smoke test)"
           command: |
             cd examples/vue-kitchen-sink
-            yarn storybook
-          background: true
+            yarn storybook --smoke-test
       - run:
-          name: "Run angular-cli"
+          name: "Run angular-cli (smoke test)"
           command: |
             cd examples/angular-cli
-            yarn storybook
-          background: true
+            yarn storybook --smoke-test
       - run:
-          name: Workaround for https://github.com/GoogleChrome/puppeteer/issues/290
-          command: sh ./scripts/workaround-puppeteer-issue-290.sh
+          name: "Run image snapshots"
+          command: yarn test --image
```

Comment:
Please store image diffs as artifacts (see below, plus [docs](https://circleci.com/docs/2.0/artifacts/#uploading-artifacts))

That way it would be much easier to find out why is the CI test failing

Reply:
Nice, will do !